USER_MANUAL.md

```diff
@@ -1549,58 +1740,983 @@ The following figure offers a visual guideline for the various parameters that c
+
+Only the [equipment](#equipment-equipment-file) column is mandatory.
+
+Potential columns are:
+
+- [{component}](#component-equipment-file)
+- [repairable_repair_delay](#repairable_repair_delay-equipment-file)
+- [repairable_repair_cost](#repairable_repair_cost-equipment-file)
+- [repairable_emissions_rate_source](#repairable_emissions_rate_source-equipment-file)
+- [repairable_emissions_production_rate](#repairable_emissions_production_rate-equipment-file)
+- [repairable_duration](#repairable_duration-equipment-file)
+- [repairable_multiple_emissions_per_source](#repairable_multiple_emissions_per_source-equipment-file)
+- [non_repairable_emissions_rate_source](#non_repairable_emissions_rate_source-equipment-file)
+- [non_repairable_emissions_production_rate](#non_repairable_emissions_production_rate-equipment-file)
+- [non_repairable_duration](#non_repairable_duration-equipment-file)
+- [non_repairable_multiple_emissions_per_source](#non_repairable_multiple_emissions_per_source-equipment-file)
+
+Method-specific columns are values that can be defined for a particular method. To use them, replace the `{method}` placeholder with the relevant [method_name](#method_name) that the user is parameterizing. The following method-specific column is available for the equipment file:
+
+- {method}_[spatial](#method_spatial-equipment-file)
+
+**Note:** As with most name-related parameters in LDAR-Sim, the values in the `equipment` and the user defined _component_ columns are case-sensitive and must be consistent across all related files. Any method specific parameters also require the same care with the case-sensitivity and consistency in all relevant files with the method name.
+
+See [equipment file](#equipment-file) for details on how to set the parameter for simulation.
+
+--------------------------------------------------------------------------------
+
+Below is an example of an _equipment file_ that defines 3 different groups:
+
+| equipment |flare|tank|pump|
+|----|----|----|----|
+|group1|1|1|0|
+|group2|0|0|2|
+|group3|3|0|0|
+
+--------------------------------------------------------------------------------
+
+#### equipment (Equipment File)
+
+**Description:** A user-defined value that defines a specific equipment group for the simulation. These equipment groups influence how  `equipment level` [measurement scale](#measurement_scale)  methods detect and measure emissions.
+
+**Note:** It is case-sensitive and must remain consistent when provided in other infrastructure files, such as the [sites file](#sites-file) or the [site type file](#site-type-file)
+
+#### {component} (Equipment File)
+
+**Description:** The column header is a user defined value of a specific component in the simulation. The values in each column and row indicate how many counts of the given component(column) exists for the given equipment(row). These _equipment_ groups can serve as building blocks for constructing complex sites.
```

Comment:
Nit: The column header is a user defined value corresponding the identifier of a specific component defining as part of the simulation.